src/uiTest/scala/com/virtuslab/gitmachete/uitest/UISuite.scala

```diff
@@ -27,4 +27,9 @@ trait UISuite extends RunningIntelliJPerSuite with IdeProbeFixture with RunningI
     intelliJ.ide.closeOpenedProjects()
   }
 
+  def doAndAwait(action: => Unit): Unit = {
```

Comment:
Hmmm does it need to be defined both here and in src/uiTest/scala/com/virtuslab/gitmachete/uitest/RunningIntelliJFixtureExtension.scala ? 🤔